### ISSUE TITLE
add missing scripts

### ISSUE THREAD

**Issue by nicolerg** [NikVetr/MoTrPAC_Complex_Traits#1]
Can these scripts be added to this repo? https://github.com/NikVetr/MoTrPAC_Complex_Traits/blob/main/fig1b_integration-with-metaanalysis.R#L26
```r
#load de data
source(file = "~/scripts/montgomery_lab/load_deg-eqtl_merged_file.R")

#load functions
source(file = "~/scripts/montgomery_lab/deg-trait_functions.R")
```

**Comment by nicolerg** [comment]
These scripts are sourced in several places, so they should be added to the repo, and the line should be replaced with the new relative path

**Comment by NikVetr** [comment]
yes! thank you for poking through, gonna return to tidying everything next
